core/src/main/groovy/com/novoda/gradle/release/AndroidArtifacts.groovy

```diff
@@ -1,42 +1,48 @@
 package com.novoda.gradle.release
 
 import org.gradle.api.Project
+import org.gradle.api.Task
+import org.gradle.api.component.SoftwareComponent
 import org.gradle.api.tasks.bundling.Jar
 import org.gradle.api.tasks.javadoc.Javadoc
 
 class AndroidArtifacts implements Artifacts {
 
     def variant
 
+    // TODO: Declare that variant is
+    // https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.LibraryExtension.html#com.android.build.gradle.LibraryExtension:libraryVariants
```

Comment:
how would this work?

Reply:
I believe this is done on purpose to avoid to have the plugin to compile against the Android Gradle Plugin and leverage instead the dynamicity of Groovy to play it's magic at runtime. I still believe is a better option because altho the IDE support is not great we won't be tying our plugin to any specific version of the plugin. 
So yeah, I would remove that `TODO` :)

Reply:
Yep. It is exactly like @mr-archano said.
You are right, just removed the todo 👍